test/add-member.test.js

```diff
@@ -10,8 +10,8 @@ const Testbot = require('./helpers/testbot')
 const replicate = require('./helpers/replicate')
 
 test('get added to a group', async (t) => {
-  const alice = Testbot({ keys: ssbKeys.generate(null, 'alice') })
-  const bob = Testbot({ keys: ssbKeys.generate(null, 'bob') })
+  const alice = Testbot({ keys: ssbKeys.generate(null, 'alice', 'buttwoo-v1') })
+  const bob = Testbot({ keys: ssbKeys.generate(null, 'bob', 'buttwoo-v1') })
```

Comment:
This is going to get quite complicated soon when we add ssb-meta-feeds to ssb-tribes2 and then it'll try to declare the buttwoo feed as the "main". Also, the fact this works now with shs is coincidence, and there may be a bunch of other places where this could break because the `@` sigil feeds are conflated with "network identity" in a bunch of places.

I recommend reverting this change, and also we don't need loadOrCreate to support other feed formats. We're going to live with classic sigil IDs as the "network identity" until the root metafeed is the primary identity, and until we start supporting https://github.com/ssbc/ssb-network-identity-spec .

Reply:
Yes this is more of a cheap and simple way to simulate metafeeds